ci/unit_tests/workspace_addjson/main_test.py

```diff
@@ -0,0 +1,32 @@
+
+import os, unittest, sys, codecs, json
+
+import workspace_addjson
+
+class TestMain(unittest.TestCase):
+
+    ''' Tests if particular file is found by getFilesAtPath function '''
```

Comment:
Please update doc comment and put it inside the function.

Reply:
done